server/app/middlewares/auth.js

```diff
@@ -0,0 +1,37 @@
+import validator from 'email-validator';
```

Comment:
Please write your own custom validator. This package flouts the rule that no packages should be used.

Reply:
done